dev/ansible/00_generate_secrets_vault.yml

```diff
@@ -1,11 +1,26 @@
 - name: Generate random strings to use for or presto cluster communication
   set_fact:
     cluster_shared_secret: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=12') }}"
+    minio_access_key: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=12') }}"
+    minio_secret_key: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=12') }}"
+    pg_username: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=12') }}"
+    pg_password: "{{ lookup('password', '/dev/null chars=ascii_lowercase,digits length=12') }}"
+
+- name: Put credentials for minio in vault k/v
+  shell: vault kv put secret/dev/minio access_key="{{ minio_access_key }}" secret_key="{{ minio_secret_key }}"
+  run_once: true
+  environment:
+    VAULT_TOKEN: "{{ lookup('env', 'vault_master_token') }}"
```

Comment:
Do we want this to run no matter what? Wouldn't it make more sense if it only runs if that specific service uses vault?

Reply:
I want to avoid creating one more switch or variable for it. So, let's run it no matter what. (both modes `cluster` and `standalone` can be run with Vault provided creds)